packages/kbn-pm/src/utils/project.ts

```diff
@@ -157,7 +157,7 @@ export class Project {
     return runScriptInPackage(scriptName, args, this);
   }
 
-  async runScriptStreaming(scriptName: string, args: string[] = []) {
+  runScriptStreaming(scriptName: string, args: string[] = []) {
```

Comment:
__note:__ basically reverting to the signature that was before TS migration to have access to the stream itself.